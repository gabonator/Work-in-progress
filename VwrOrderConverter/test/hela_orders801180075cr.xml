
<?xml version="1.0" encoding="windows-1250"?>
<biztalk_1><header><delivery><message><messageID></messageID><sent></sent>
<subject></subject></message><to><address/><state><referenceID></reference
ID><handle></handle><process></process><ediid>63073242</ediid></state></to
><from><address/><state><referenceID></referenceID><handle></handle><proce
ss></process><ediid>25963961</ediid></state></from></delivery><manifest><d
ocument><userdocid>801180075</userdocid><name>xml_document</name></documen
t></manifest></header><body><xml_document
xmlns="http://editel.cz/schemas/2002/orders.xdr"><form_header><userdocid>8
01180075</userdocid><document_date>2018-02-15</document_date><delivery_dat
e>1900-01-01</delivery_date><buyer_ean>25963961</buyer_ean><buyer_name>HEL
AGO-CZ,
s.r.o.</buyer_name><delivery_ean>63073242</delivery_ean><delivery_name>VWR
International
s.r.o.</delivery_name><supplier_ean>63073242</supplier_ean><supplier_name>
VWR International
s.r.o.</supplier_name></form_header><line_items><item><item_number>1</item
_number><article_ean>331355060500</article_ean><article_name>St
bezpe
500</article_name><quantity>10</quantity><unit>ks</unit><specification>St
i
500</specification></item><item><item_number>2</item_number><article_ean>3
31355030250</article_ean><article_name>St
LDPE, s potiskem, Isopropanol, 250
m</article_name><quantity>10</quantity><unit>ks</unit><specification>St
ka bezpe
m</specification></item><item><item_number>3</item_number><article_ean>331
707132605</article_ean><article_name>St
�zkohrdl�, zelen�,
250ml</article_name><quantity>10</quantity><unit>ks</unit><specification>S
t�i�ka celobarevn�, PE-LD, �zkohrdl�, zelen�,
250ml</specification></item><item><item_number>4</item_number><article_ean
>331707132603</article_ean><article_name>St�i�ka celobarevn�, PE-LD,
�zkohrdl�, �erven�,250
ml</article_name><quantity>10</quantity><unit>ks</unit><specification>St
�ka celobarevn�, PE-LD, �zkohrdl�, �erven�,250
ml</specification></item></line_items></xml_document></body></biztalk_1>
