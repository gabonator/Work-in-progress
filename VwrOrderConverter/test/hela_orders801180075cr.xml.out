BEGIN;801180075;HELAGO-CZ, s.r.o.;VWR;ORDERS;20180215;102511;HEAD;801180075;20180215;;CZK;SOLDTO;26000611;HELAGO-CZ, s.r.o.SHIPTO;26000611;HELAGO-CZ, s.r.o.;;;;Kladská 1082;;;;;Hradec Králové 3;500 03;CZ;odbyt@helago-cz.cz;495 768 317LINE;000001;;331355060500.;Střička bezpečnostní širokohrdlá, LDPE, s potiskem, destil. voda, 500;10.000;;20180215;;LINE;000002;;331355030250.;Střička bezpečnostní širokohrdlá, LDPE, s potiskem, Isopropanol, 250 m;10.000;;20180215;;LINE;000003;;331707132605.;Střička celobarevná, PE-LD, úzkohrdlá, zelená, 250ml;10.000;;20180215;;LINE;000004;;331707132603.;Střička celobarevná, PE-LD, úzkohrdlá, červená,250 ml;10.000;;20180215;;END;801180075